packages/core/src/app/shipping/hooks/useMultishippingConsignmentItems.ts

```diff
@@ -0,0 +1,110 @@
+import { Consignment, LineItemMap } from "@bigcommerce/checkout-sdk";
+
+import { useCheckout } from "@bigcommerce/checkout/payment-integration-api";
+
+import { LineItemType, MappedDataConsignment, MultiShippingTableItemWithType, UnassignedItems } from "../MultishippingV2Type";
+
+interface MultiShippingConsignmentItemsHook {
+    unassignedItems: UnassignedItems;
```

Comment:
Can we have a different name for interface? Unassigned items seems very specific.